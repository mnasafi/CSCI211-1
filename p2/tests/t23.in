title
title 00
url 00
comment 00
55.87
1
title 01
url 01
comment 01
3.83
5
title 02
url 02
comment 02
11.39
2
title 03
url 03
comment 03
51.44
3
title 04
url 04
comment 04
79.47
1
title 05
url 05
comment 05
52.74
2
title 06
url 06
comment 06
93.3
4
title 07
url 07
comment 07
15.32
4
title 08
url 08
comment 08
44.66
1
title 09
url 09
comment 09
2.35
3
title 10
url 10
comment 10
42.64
2
title 11
url 11
comment 11
77.71
2
title 12
url 12
comment 12
17.56
2
title 13
url 13
comment 13
13.42
4
title 14
url 14
comment 14
61.76
2
title 15
url 15
comment 15
87.3
5
title 16
url 16
comment 16
42.57
4
title 17
url 17
comment 17
74.5
3
title 18
url 18
comment 18
92.51
3
title 19
url 19
comment 19
27.28
2
title 20
url 20
comment 20
92.5
3
title 21
url 21
comment 21
63.37
4
title 22
url 22
comment 22
71.75
2
title 23
url 23
comment 23
13.71
5
title 24
url 24
comment 24
41.21
2
title 25
url 25
comment 25
45.49
5
title 26
url 26
comment 26
2.28
1
title 27
url 27
comment 27
7.3
1
title 28
url 28
comment 28
58.92
2
title 29
url 29
comment 29
38.24
3
title 30
url 30
comment 30
44.62
1
title 31
url 31
comment 31
33.41
1
title 32
url 32
comment 32
8.97
1
title 33
url 33
comment 33
79.68
1
title 34
url 34
comment 34
52.4
1
title 35
url 35
comment 35
1.52
5
title 36
url 36
comment 36
81.86
1
title 37
url 37
comment 37
84.13
5
title 38
url 38
comment 38
76.13
4
title 39
url 39
comment 39
52.4
2
title 40
url 40
comment 40
15.74
5
title 41
url 41
comment 41
56.34
2
title 42
url 42
comment 42
53.61
4
title 43
url 43
comment 43
74.7
4
title 44
url 44
comment 44
30.69
2
title 45
url 45
comment 45
83.19
2
title 46
url 46
comment 46
21.66
2
title 47
url 47
comment 47
87.61
2
title 48
url 48
comment 48
100.67
2
title 49
url 49
comment 49
57.31
3
title 50
url 50
comment 50
83.32
2
title 51
url 51
comment 51
69.28
5
title 52
url 52
comment 52
31.56
4
title 53
url 53
comment 53
90.91
4
title 54
url 54
comment 54
59.4
4
title 55
url 55
comment 55
30.20
3
title 56
url 56
comment 56
96.88
5
title 57
url 57
comment 57
10.36
1
title 58
url 58
comment 58
77.67
3
title 59
url 59
comment 59
8.73
4
title 60
url 60
comment 60
92.82
3
title 61
url 61
comment 61
72.6
1
title 62
url 62
comment 62
28.85
1
title 63
url 63
comment 63
19.20
5
title 64
url 64
comment 64
75.29
5
title 65
url 65
comment 65
47.19
5
title 66
url 66
comment 66
35.43
5
title 67
url 67
comment 67
71.87
1
title 68
url 68
comment 68
90.18
3
title 69
url 69
comment 69
63.52
2
title 70
url 70
comment 70
46.42
1
title 71
url 71
comment 71
4.14
2
title 72
url 72
comment 72
89.54
2
title 73
url 73
comment 73
61.77
2
title 74
url 74
comment 74
42.36
2
title 75
url 75
comment 75
61.58
5
title 76
url 76
comment 76
56.57
2
title 77
url 77
comment 77
44.3
2
title 78
url 78
comment 78
14.11
2
title 79
url 79
comment 79
66.43
1
title 80
url 80
comment 80
8.34
3
title 81
url 81
comment 81
22.21
1
title 82
url 82
comment 82
28.80
1
title 83
url 83
comment 83
5.66
4
title 84
url 84
comment 84
94.74
5
title 85
url 85
comment 85
52.57
3
title 86
url 86
comment 86
61.87
3
title 87
url 87
comment 87
16.3
1
title 88
url 88
comment 88
27.41
1
title 89
url 89
comment 89
22.69
1
title 90
url 90
comment 90
9.12
5
title 91
url 91
comment 91
30.99
3
title 92
url 92
comment 92
10.42
1
title 93
url 93
comment 93
28.24
5
title 94
url 94
comment 94
3.67
5
title 95
url 95
comment 95
60.52
1
title 96
url 96
comment 96
47.49
3
title 97
url 97
comment 97
2.53
4
title 98
url 98
comment 98
44.1
2
title 99
url 99
comment 99
13.61
4
