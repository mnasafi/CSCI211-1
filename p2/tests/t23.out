title 00, url 00, comment 00, 55.87, *
title 01, url 01, comment 01, 3.83, *****
title 02, url 02, comment 02, 11.39, **
title 03, url 03, comment 03, 51.44, ***
title 04, url 04, comment 04, 79.47, *
title 05, url 05, comment 05, 52.74, **
title 06, url 06, comment 06, 93.3, ****
title 07, url 07, comment 07, 15.32, ****
title 08, url 08, comment 08, 44.66, *
title 09, url 09, comment 09, 2.35, ***
title 10, url 10, comment 10, 42.64, **
title 11, url 11, comment 11, 77.71, **
title 12, url 12, comment 12, 17.56, **
title 13, url 13, comment 13, 13.42, ****
title 14, url 14, comment 14, 61.76, **
title 15, url 15, comment 15, 87.3, *****
title 16, url 16, comment 16, 42.57, ****
title 17, url 17, comment 17, 74.5, ***
title 18, url 18, comment 18, 92.51, ***
title 19, url 19, comment 19, 27.28, **
title 20, url 20, comment 20, 92.5, ***
title 21, url 21, comment 21, 63.37, ****
title 22, url 22, comment 22, 71.75, **
title 23, url 23, comment 23, 13.71, *****
title 24, url 24, comment 24, 41.21, **
title 25, url 25, comment 25, 45.49, *****
title 26, url 26, comment 26, 2.28, *
title 27, url 27, comment 27, 7.3, *
title 28, url 28, comment 28, 58.92, **
title 29, url 29, comment 29, 38.24, ***
title 30, url 30, comment 30, 44.62, *
title 31, url 31, comment 31, 33.41, *
title 32, url 32, comment 32, 8.97, *
title 33, url 33, comment 33, 79.68, *
title 34, url 34, comment 34, 52.4, *
title 35, url 35, comment 35, 1.52, *****
title 36, url 36, comment 36, 81.86, *
title 37, url 37, comment 37, 84.13, *****
title 38, url 38, comment 38, 76.13, ****
title 39, url 39, comment 39, 52.4, **
title 40, url 40, comment 40, 15.74, *****
title 41, url 41, comment 41, 56.34, **
title 42, url 42, comment 42, 53.61, ****
title 43, url 43, comment 43, 74.7, ****
title 44, url 44, comment 44, 30.69, **
title 45, url 45, comment 45, 83.19, **
title 46, url 46, comment 46, 21.66, **
title 47, url 47, comment 47, 87.61, **
title 48, url 48, comment 48, 100.67, **
title 49, url 49, comment 49, 57.31, ***
title 50, url 50, comment 50, 83.32, **
title 51, url 51, comment 51, 69.28, *****
title 52, url 52, comment 52, 31.56, ****
title 53, url 53, comment 53, 90.91, ****
title 54, url 54, comment 54, 59.4, ****
title 55, url 55, comment 55, 30.2, ***
title 56, url 56, comment 56, 96.88, *****
title 57, url 57, comment 57, 10.36, *
title 58, url 58, comment 58, 77.67, ***
title 59, url 59, comment 59, 8.73, ****
title 60, url 60, comment 60, 92.82, ***
title 61, url 61, comment 61, 72.6, *
title 62, url 62, comment 62, 28.85, *
title 63, url 63, comment 63, 19.2, *****
title 64, url 64, comment 64, 75.29, *****
title 65, url 65, comment 65, 47.19, *****
title 66, url 66, comment 66, 35.43, *****
title 67, url 67, comment 67, 71.87, *
title 68, url 68, comment 68, 90.18, ***
title 69, url 69, comment 69, 63.52, **
title 70, url 70, comment 70, 46.42, *
title 71, url 71, comment 71, 4.14, **
title 72, url 72, comment 72, 89.54, **
title 73, url 73, comment 73, 61.77, **
title 74, url 74, comment 74, 42.36, **
title 75, url 75, comment 75, 61.58, *****
title 76, url 76, comment 76, 56.57, **
title 77, url 77, comment 77, 44.3, **
title 78, url 78, comment 78, 14.11, **
title 79, url 79, comment 79, 66.43, *
title 80, url 80, comment 80, 8.34, ***
title 81, url 81, comment 81, 22.21, *
title 82, url 82, comment 82, 28.8, *
title 83, url 83, comment 83, 5.66, ****
title 84, url 84, comment 84, 94.74, *****
title 85, url 85, comment 85, 52.57, ***
title 86, url 86, comment 86, 61.87, ***
title 87, url 87, comment 87, 16.3, *
title 88, url 88, comment 88, 27.41, *
title 89, url 89, comment 89, 22.69, *
title 90, url 90, comment 90, 9.12, *****
title 91, url 91, comment 91, 30.99, ***
title 92, url 92, comment 92, 10.42, *
title 93, url 93, comment 93, 28.24, *****
title 94, url 94, comment 94, 3.67, *****
title 95, url 95, comment 95, 60.52, *
title 96, url 96, comment 96, 47.49, ***
title 97, url 97, comment 97, 2.53, ****
title 98, url 98, comment 98, 44.1, **
title 99, url 99, comment 99, 13.61, ****
